meetings/2022-11-17.md

```diff
@@ -0,0 +1,128 @@
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Sarven
+
+
+### Introductions
+
+* name: text
+
+---
+
+## Topics
+
+* SC: Keeping the topic focus to work we need to iron out for the next release.
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback. Links to servers/applications and demos welcome.
+* eP: webhooks should land in CSS 6.0 https://github.com/CommunitySolidServer/CommunitySolidServer/pull/1388#issuecomment-1318295827
+* SC: We should discuss WH during next meeting maybe invite Jackson
+
+ACTION: eP to reach out to Jackson
+
+### Release of the spec
+URL: https://github.com/solid/notifications-panel/blob/main/meetings/2022-11-10.md#release-of-the-spec
+
+>ACTION: SC to review https://hackmd.io/3sLhQvmKTVKQKoXeXLSTpA
+>ACTION: SC to create a project board for the releases.
+
+* SC: Still a todo.
+
+
+### Update Notifications UCR
+URL: https://github.com/solid/notifications-panel/issues/76
+
+>SC: I'll review https://github.com/solid/notifications-panel/pull/79
+
+* SC: In the middle of reviewing.
+* SC: Checking to see if one is covered by existing. Not sure yet.
+* RG: There are multiple design choices. Making UCs explicit constraints the design space.
+* eP: Seems like reasonable UCs we can iterate on.
+
+ACTION : eP to review.
+
+* eP: Approved.
+* SC: https://github.com/solid/notifications-panel/issues/76 will focus on editorial and other iterations.
+* eP: Some of them may be about managing connections. That may be too much for the use case but use connections for the same client or subscriber, or for multiple connections. Especially for websockets the server may want to reuse same connection for the same client. This sounds like a requirement for me no matter how exactly we solve it.
+* eP: Which parts of these go into the UC or too much detail for the UC.
+* RG: More like requirement than a UC document based on real-world stuff. We need the practical UCs and then the requirements out of that. Based on that we need to justify some design decisions.
+
+
+### Simplify flow diagrams
+URL: https://github.com/solid/notifications/pull/108
+
+* SC: Any objections to merge?
+* eP: It is still approved from me and has requested changes from Sarven
+* SC: I see it is the loop part. No objections to merge it with it in.
+* SC: Rahul are you ok with this?
+* RG: +1 I'm good with it.
+* SC: Merging
+
+
+### Considerations
+
+* SC: We may be able to process these security considerations quickly.
+* RG: You need to explain what is the context here.
+
+
+#### Add consider-exposing-information
+URL: https://github.com/solid/notifications/pull/117
+
+* SC: Exposing information advises to avoid exposing any information that is not required for the interaction. For example when notification is provided, it may include which resource was changed but not which agent changed it. We don't want servers to just slip in such information.
```

Comment:
```suggestion
* SC: Exposing information advises to avoid exposing any information that is not required for the interaction. For example, when notification is provided, it may include which resource was changed but not which agent changed it. We don't want servers to just slip in such information.
```